lib/ts/frameworks/awsLambda.ts

```diff
@@ -0,0 +1,81 @@
+/* Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
```

Comment:
Should the folder be called `"framework"` instead of `"frameworks"`? Cause everywhere else you use `framework`